web-client/src/presenter/actions/setTokenAction.test.js

```diff
@@ -55,24 +55,4 @@ describe('setTokenAction,', () => {
       mockToken,
     );
   });
-
-  it('should call applicationContext.getUseCases().setItemInteractor with key token and the value of props.token', async () => {
```

Comment:
Instead of removing this, could we setup process.env.IS_LOCAL for this test so that code is still covered?